docs/src/components/Layout/GlobalNav/GlobalNav.module.scss

```diff
@@ -0,0 +1,157 @@
+  --amplify-colors-font-hover: #000;
+}
+
+.navbar {
+  box-shadow: 0px 4px 16px 0px rgba(0, 0, 0, 0.04);
+  border-bottom: 1px solid #d5dbdb;
+  background-color: var(--amplify-colors-background-primary);
+  position: sticky;
+  top: 0px;
+  z-index: 200;
+
+  a,
+  p,
+  span {
+    font-family: 'Amazon Ember' !important;
+    -webkit-font-smoothing: auto !important;
+    -moz-osx-font-smoothing: auto !important;
+  }
+
+  --dev-center-nav-height: 60px;
+  --amplify-colors-font-hover: #000;
+}
+
+[data-amplify-theme='amplify-docs'][data-amplify-color-mode='dark'] .navbar {
+  --amplify-colors-font-hover: inherit;
+}
+
+[data-amplify-theme='amplify-docs'][data-amplify-color-mode='system'] .navbar {
+  @media (prefers-color-scheme: dark) {
+    --amplify-colors-font-hover: inherit;
+  }
+}
+
+.dev-center-logo {
+  color: var(--amplify-colors-font-primary);
+  font-size: 1.25rem;
+  white-space: nowrap;
+}
+
+.nav-container {
+  display: flex;
+  height: var(--dev-center-nav-height);
+  align-items: center;
+  justify-content: space-between;
+}
+
+.mobile-nav-container {
+  display: flex;
+  height: var(--dev-center-nav-height);
+  align-items: center;
+  justify-content: space-between;
+  padding: 0px 18px;
+}
+
+.nav-links-container {
+  height: var(--dev-center-nav-height);
+  align-items: center;
+  justify-content: space-between;
+}
+
+.mobile-nav-menu-container {
+  z-index: 1000;
+  background-color: var(--amplify-colors-background-primary);
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-bottom: 1px solid #d5dbdb;
+}
+
+.mobile-nav-icons-container {
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+
+  svg {
+    display: block;
+    margin: auto;
+  }
+}
+
+.mobile-nav-menu-items {
+  border-top: 1px solid #d5dbdb;
+  height: 48px;
+  width: 100%;
+}
+
+.nav-menu-item {
+  padding: 12px;
+  display: flex;
+  align-items: center;
+  height: 100%;
+  font-size: 1rem;
+  text-decoration: none;
+  color: var(--amplify-colors-font-tertiary) !important;
+  font-family: 'Amazon Ember';
+  cursor: pointer;
+  font-weight: 400;
+
+  white-space: nowrap;
+
+  &:hover,
+  &:focus {
+    color: var(--amplify-colors-font-hover) !important;
+
+    .custom-path-stroke {
+      stroke: var(--amplify-colors-font-hover) !important;
+    }
+  }
+
+  .custom-path-stroke {
+    stroke: var(--amplify-colors-font-tertiary);
+  }
+}
+
+.secondary-nav-button {
+  border-top: 1px solid #d5dbdb;
+  border-bottom: 1px solid #d5dbdb;
+
+  // Set color for back chevron icon
+  color: var(--amplify-colors-font-tertiary);
+
+  p {
+    color: var(--amplify-colors-font-tertiary);
+  }
+
+  &:hover {
+    color: black;
+    border: 1px solid #d5dbdb;
+    background-color: white !important;
+
+    p {
+      color: black;
+    }
+  }
+}
+
+.current-nav-menu-item {
+  border-top: 2px solid transparent;
+  border-bottom: 2px solid #ff9900;
```

Comment:
Right now the border for the active tab sits on top of the border for the nav bar. Usually, the active indicator doesn't sit on top, but above the border for the tab bar. We can fix this by giving a negative bottom margin of 1px.